projects/packages/forms/src/blocks/input/edit.js

```diff
@@ -31,12 +32,16 @@ const getInputClass = type => {
 const InputEdit = ( { attributes, clientId, isSelected, name, setAttributes, context } ) => {
 	const { 'jetpack/field-share-attributes': isSynced } = context;
 	useSyncedAttributes( name, isSynced, SYNCED_ATTRIBUTE_KEYS, attributes, setAttributes );
-
 	const { max, min, placeholder, type } = attributes;
+	const styles = useFormStyleOutlineClassesAndStyles( {
+		clientId,
+		inputBlockName: name,
+		inputBlockAttributes: attributes,
+	} );
```

Comment:
Nice! I've just been toying with multiple options. 

I think they need a variation of the new `useSiblingBlock` hook in label, but to get the attributes of the `jetpack/options` field. E.g., 

```js
function useInnerOptionsBlock( clientId ) {
	const inputBlock = useSelect(
		select => {
			// Get the parent block
			const parentBlock = select( blockEditorStore ).getBlock( clientId );
			if ( ! parentBlock ) {
				return {};
			}

			return parentBlock.innerBlocks.find( block => block.name === 'jetpack/options' );
		},
		[ clientId ]
	);

	return inputBlock;
}
```

Looking at this now.